content.js

```diff
@@ -49,6 +55,49 @@ async function copyPrDescription() {
   messageFields.forEach(f => f.value = commitBody);
 }
 
+async function getPrMetadataFromDocument() {
+  await makePrDescriptionAppear();
+
+  const prTitleEl = document.getElementById('issue_title');
+  if (!prTitleEl) {
+    warn('failed to find PR title element');
+    return null;
+  }
+
+  let prBodyEl = document.querySelector('.comment-form-textarea[name="issue[body]"]');
+  if (!prBodyEl) {
+    prBodyEl = document.querySelector('.comment-form-textarea[name="pull_request[body]"]');
+    if (!prBodyEl) {
+      warn('failed to find PR body element');
+      return null;
+    }
+  }
+
+  return {
+    title: prTitleEl.value,
+    body: prBodyEl.textContent,
+  };
+}
+
+async function getPrMetadataFromApi(repo, prNumber) {
+  const response = await fetch(`https://api.github.com/repos/${repo}/pulls/${prNumber}`);
```

Comment:
Should we check for error here? or perhaps only run this if we know the site is github.com, because we know this is not going to work in Enterprise or Private repos.

You can use `window.location.host` to check if the site is `github.com`

Reply:
Since we're parsing location.pathname to get the repo and PR number for the parameters, I think a check at that level might be more sensible.  I'll add it there.

Reply:
Done